nox/sessions.py

```diff
@@ -286,14 +286,14 @@ def description(self):
         return None
 
     def __str__(self):
-        return self.signature or self.name
+        return self.signatures[0] if self.signatures else self.name
```

Comment:
This leaves data unused?

Reply:
Yes it makes sense, but ISTM that's a bad idea to use `__str__` for this. Why don't you have a `@property`, e.g. `Session.preferred` and make `__str__` actually be comprehensive / descriptive?

Reply:
That sounds more like a job for `__repr__`, but, if you still prefer I can do the whole property thing.